core/src/main/resources/dokka/styles/style.css

```diff
@@ -193,7 +197,7 @@ td:first-child {
 }
 
 .symbol {
-    font-family: Monaco, Bitstream Vera Sans Mono, Lucida Console, Terminal;
+    font-family: JetBrains Mono;
```

Comment:
We should have a fallback font if this one can't be found

Reply:
Added.